pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/nonpepperkit/NonPepperKitCreationService.java

```diff
@@ -98,7 +124,8 @@ private SimpleResult createKit(@NonNull DDPInstance ddpInstance, @NonNull KitTyp
             userId = JUNIPER;
         } else {
             log.warn("Seems like {} is not configured as a JUNIPER study! ", ddpInstance.getName());
-            transactionResults.resultException = new DsmInternalError(ddpInstance.getName() + " study is not configured to use this method.");
+            transactionResults.resultException =
+                    new DsmInternalError(ddpInstance.getName() + " study is not configured to use this method.");
```

Comment:
"this method" seems vague. Can you say something like "study is not configured as a Juniper study", since that seems to be the specific case here? (And matches the warning message.)